doc/source/conf.py

```diff
@@ -332,126 +182,40 @@
 \titlespacing*{\paragraph}{0pt}{1ex}{0pt}
 \makeatother
 
-'''
-
-# Documents to append as an appendix to all manuals.
-#latex_appendices = []
-
-# If false, no module index is generated.
+"""
 latex_domain_indices = False
 
-# -----------------------------------------------------------------------------
+# ----------------------------------------------------------------------------
 # Numpy extensions
 # -----------------------------------------------------------------------------
 numpydoc_show_class_members = False
 numpydoc_class_members_toctree = False
 
-# -----------------------------------------------------------------------------
-# Plots
```

Comment:
Is this entire "Plots" section unused?

Reply:
It was leftover from the days before sphinx-gallery when you had your own custom gallery type thing. It was mistakenly used in one place (https://github.com/scikit-image/scikit-image/pull/6579/files#diff-efb33bcca64dab5e4915f762715f98acee535a251281448e1f2b269583407d80). If you look at the docs
- https://scikit-image.org/docs/0.19.x/api/skimage.transform.html?highlight=hough%20transform#skimage.transform.hough_line
you will notice that it includes both the old plots example as well as an improved version that is part of the sphinx gallery.

Reply:
![screenshot](https://user-images.githubusercontent.com/123428/195881675-64f49ede-b07e-44f2-8258-c78ebbf39cad.png)

Reply:
Ah, happy to see this go. :ok_hand: